src/view/com/modals/UserAddRemoveLists.tsx

```diff
@@ -182,7 +187,7 @@ function ListItem({
           <ActivityIndicator />
         ) : (
           <Button
-            testID={`user-${subject}-addBtn`}
+            testID={`user-${handle}-addBtn`}
```

Comment:
Didn't know how to get a match here aside from passing in the handle, seems innocuous enough